components/jeune/DetailsJeune.tsx

```diff
@@ -57,6 +112,39 @@ export const DetailsJeune = ({
               <dd className='text-primary'>{jeune.email}</dd>
             </div>
           )}
+
+          {structureConseiller !== StructureConseiller.MILO && (
+            <div className='flex'>
+              <dt className='text-base-regular'>Identifiant Pôle Emploi :</dt>
+              <dd
+                className='text-base-bold ml-1'
+                onCopy={trackEventOnCopieIdentifiantPartenaire}
+              >
+                {identifiantPartenaire ? identifiantPartenaire : '-'}
+              </dd>
+              <button
+                className='ml-5 flex items-center text-primary'
+                aria-label={
+                  identifiantPartenaire
+                    ? 'Modifier l’identifiant pôle emploi'
+                    : 'Ajouter l’identifiant pôle emploi'
+                }
+                onClick={openIdentifiantPartenaireModal}
+              >
+                <IconComponent
+                  name={IconName.Pen}
+                  aria-label={
+                    identifiantPartenaire
+                      ? 'Modifier l’identifiant pôle emploi'
+                      : 'Ajouter l’identifiant pôle emploi'
+                  }
+                  className='w-[11px] h-[11px] mr-1'
```

Comment:
```suggestion
                  className='w-4 h-4 mr-1'
```
source de vérité: le lama system, en général on a seulement 2 tailles possibles pour les icônes, 16px et 24px
https://pass-emploi.zeroheight.com/styleguide/s/75590/p/92f5c8-icons